CMakeLists.txt

```diff
@@ -109,6 +109,8 @@ else()
   )
 endif()
 set(AMDGPU_TARGETS "${DEFAULT_AMDGPU_TARGETS}" CACHE STRING "List of specific machine types for library to target")
```

Comment:
I guess we can also remove AMDGPU_TARGETS?

Reply:
I think we are supposed to continue supporting AMDGPU_TARGETS but just issue a deprecation warning saying to use GPU_TARGETS. I think we want to add something like what rocblas did:

if( AMDGPU_TARGETS AND NOT GPU_TARGETS )
  message( DEPRECATION "AMDGPU_TARGETS use is deprecated. Use GPU_TARGETS." )
  set( GFX_LIST ${AMDGPU_TARGETS} )
elseif( GPU_TARGETS )
  set( GFX_LIST ${GPU_TARGETS} )
else ()
  set( GFX_LIST ${DEFAULT_AMDGPU_TARGETS} ) # default if not set
endif()

# Don't force, as users should be able to override GPU_TARGETS at the command line if desired
set(GPU_TARGETS "${GFX_LIST}" CACHE STRING "GPU architectures to build for")